lib/fog/google/models/pubsub/subscription.rb

```diff
@@ -0,0 +1,90 @@
+require "base64"
+require "fog/core/model"
+require "fog/google/models/pubsub/received_message"
```

Comment:
You might not need those extra requires here. Could you confirm you need them please?